examples/resources/stackit_security_group_rule/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "stackit_security_group_rule" "example" {
```

Comment:
it would be nice to add examples for the object fields that define a rule, otherwise the example is not that useful

Reply:
added object fields in the example